aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/NetworkCapabilitiesUtilTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.amplifyframework.datastore.syncengine
+
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertFalse
+import org.junit.Test
+import org.mockito.Mockito
+
+class NetworkCapabilitiesUtilTest {
+
+    @Test
+    fun testGetNetworkCapabilitiesSecurityException() {
+        val mockConnectivityManager = Mockito.mock(ConnectivityManager::class.java)
+        Mockito.`when`(mockConnectivityManager.activeNetwork).thenThrow(SecurityException())
```

Comment:
In Kotlin tests we should use `mockk`, not `Mockito`.

```suggestion
        val mockConnectivityManager = mockk<ConnectivityManager> {
            every { activeNetwork } throws SecurityException()
        }
        Mockito.`when`(mockConnectivityManager.activeNetwork).thenThrow(SecurityException())
```